imageroot/actions/configure-module/validate-input.json

```diff
@@ -1,8 +1,42 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "configure-module input",
-    "$id": "http://schema.nethserver.org/collabora/configure-module-input.json",
+    "title": "Configure collabora",
+    "$id": "http://nethserver.org/json-schema/task/input/collabora/configure",
     "description": "Configure collabora",
-    "examples": [],
-    "type": "object"
-}
\ No newline at end of file
+    "examples": [
+        {
+            "host": "collabora.nethserver.org",
+            "http2https": true,
+            "lets_encrypt": true,
+            "admin_password": "password"
+        }
+    ],
+    "type": "object",
+    "required": [
+        "host",
+        "http2https",
+        "lets_encrypt",
+        "admin_password"
+    ],
+    "properties": {
+        "host": {
+            "type": "string",
+            "description": "Host name for the application, like 'collabora.nethserver.org'",
+            "format": "idn-hostname"
+        },
+        "lets_encrypt": {
+            "type": "boolean",
+            "title": "Let's Encrypt certificate",
+            "description": "Request a valid Let's Encrypt certificate."
+        },
+        "http2https": {
+            "type": "boolean",
+            "title": "HTTP to HTTPS redirection",
+            "description": "Redirect all the HTTP requests to HTTPS"
+        },
+        "admin_password":{
+            "type": "string",
+            "description": "Default password for administrator user, it can be changed later"
```

Comment:
Maybe it's better to set a minimum length of 1 char.

Reply:
done